src/Microsoft.DocAsCode.Dfm/Rules/DfmIncludeBlockRule.cs

```diff
@@ -7,15 +7,78 @@ namespace Microsoft.DocAsCode.Dfm
     using System.Text.RegularExpressions;
 
     using Microsoft.DocAsCode.MarkdownLite;
+    using Microsoft.DocAsCode.MarkdownLite.Matchers;
 
     public class DfmIncludeBlockRule : IMarkdownRule
     {
+        private static readonly Matcher _IncludeMatcher =
+            Matcher.WhiteSpacesOrEmpty +
+            "[!" +
+            Matcher.CaseInsensitiveString("include") +
+            Matcher.Char('+').Maybe() +
+            Matcher.WhiteSpacesOrEmpty +
+            '[' +
+            (
+                Matcher.AnyCharNot(']').RepeatAtLeast(1) |
```

Comment:
Yes, `.RepeatAtLeast(0).ToGroup("title")`

---
In reply to: [107638237](https://github.com/dotnet/docfx/pull/1472#discussion_r107638237) [](ancestors = 107638237)